src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala

```diff
@@ -2,8 +2,28 @@ package io.sdkman.changelogs.java
 
 import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
 import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{Linux64, MacOSX, Version, Windows, Zulu, removeVersion}
 
 @ChangeLog(order = "019")
```

Comment:
This order id has any meaning?

Reply:
Yep, the order that migrations are run in if they haven't been run before.